obfs4proxy/termmon.go

```diff
@@ -120,14 +120,14 @@ func newTermMonitor() (m *termMonitor) {
 		//  * Linux - Platform specific code that should always work.
 		//  * Other U*IX - Somewhat generic code, that works unless the
 		//    parent dies before the monitor is initialized.
-		if termMonitorOSInit != nil {
-			// Errors here are non-fatal, since it might still be
-			// possible to fall back to a generic implementation.
-			if err := termMonitorOSInit(m); err == nil {
-				return
-			}
-		}
-		if runtime.GOOS != "windows" {
+		//if termMonitorOSInit != nil {
```

Comment:
Is this intentional?

Reply:
I wish I could tell you, it looks intentional to me to support running by itself on android? This is from the android work before. I can remove the commented code though to make it cleaner. Thoughts? @Abushawish

Reply:
Oh didn't realize it was there before! Probably predates my work. Let's remove it and if we ever need it we can just look back at the commit. Thanks!